src/modules/rover_pos_control/RoverAttitudeControl.cpp

```diff
@@ -0,0 +1,128 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2013-2016 Estimation and Control Library (ECL). All rights reserved.
```

Comment:
```suggestion
 *   Copyright (c) 2013-2016 Estimation and Control Library (ECL). All rights reserved.
 *   Copyright (c) 2017-2020 PX4 Development Team. All rights reserved. 
```

And is this source actually from ECL and it's now copied into the PX4 source? I'm a bit confused.

Reply:
Yes, it was moved over from https://github.com/PX4/Firmware/pull/14284